From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Bochun Bai <bbc@dongxi.ca>
Date: Mon, 15 May 2023 17:33:03 +0200
Subject: add-board-nanopi-r4se

> X-Git-Archeology: - Revision 9c4f0524e02d71ed90b3700e923c40ba54e962b6: https://github.com/armbian/build/commit/9c4f0524e02d71ed90b3700e923c40ba54e962b6
> X-Git-Archeology:   Date: Mon, 15 May 2023 17:33:03 +0200
> X-Git-Archeology:   From: Bochun Bai <bbc@dongxi.ca>
> X-Git-Archeology:   Subject: add dts patch for kernel 6.1
> X-Git-Archeology:
> X-Git-Archeology: - Revision 3cb92f620433d4f90f40dd66edbb98bc0b88b55d: https://github.com/armbian/build/commit/3cb92f620433d4f90f40dd66edbb98bc0b88b55d
> X-Git-Archeology:   Date: Mon, 15 May 2023 17:33:03 +0200
> X-Git-Archeology:   From: Bochun Bai <bbc@dongxi.ca>
> X-Git-Archeology:   Subject: add support for NanoPi R4SE
> X-Git-Archeology:
> X-Git-Archeology: - Revision 35d5ae75ff8a18a288d6b818de6344cc79566b76: https://github.com/armbian/build/commit/35d5ae75ff8a18a288d6b818de6344cc79566b76
> X-Git-Archeology:   Date: Mon, 15 May 2023 17:33:03 +0200
> X-Git-Archeology:   From: Igor <igor@armbian.com>
> X-Git-Archeology:   Subject: Add to EDGE to and enable EDGE kernel in board config
> X-Git-Archeology:
---
 arch/arm64/boot/dts/rockchip/Makefile               |  1 +
 arch/arm64/boot/dts/rockchip/rk3399-nanopi-r4se.dts | 17 ++++++++++
 2 files changed, 18 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/Makefile b/arch/arm64/boot/dts/rockchip/Makefile
index 99a44c400d6a..30e193fa5d06 100644
--- a/arch/arm64/boot/dts/rockchip/Makefile
+++ b/arch/arm64/boot/dts/rockchip/Makefile
@@ -47,6 +47,7 @@ dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-nanopi-m4b.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-nanopi-neo4.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-nanopi-r4s.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-nanopi-r4s-enterprise.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-nanopi-r4se.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-orangepi.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-pinebook-pro.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-pinephone-pro.dtb
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-nanopi-r4se.dts b/arch/arm64/boot/dts/rockchip/rk3399-nanopi-r4se.dts
new file mode 100644
index 000000000000..1c9d0c34f151
--- /dev/null
+++ b/arch/arm64/boot/dts/rockchip/rk3399-nanopi-r4se.dts
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+
+/dts-v1/;
+#include "rk3399-nanopi-r4s-enterprise.dts"
+
+/ {
+	model = "FriendlyElec NanoPi R4SE";
+	compatible = "friendlyarm,nanopi-r4se", "rockchip,rk3399";
+};
+
+&emmc_phy {
+	status = "okay";
+};
+
+&sdhci {
+	status = "okay";
+};
-- 
Armbian

